validate_asciidoc.sh

```diff
@@ -17,10 +17,15 @@ do
     fi
   else
     #validate asciidoc
-    for language in $dir/*/
+	supportedLanguages="abap apex cfamily cobol csharp css flex go html java javascript kotlin php pli plsql python rpg ruby rust scala solidity swift tsql vb6 vbnet xml"
```

Comment:
```suggestion
	supportedLanguages=$(sed 's/ or//' supported_languages.adoc | tr -d '`,')
```
To reduce duplication where we can :)

Reply:
Great! I did not know how to do that...